docs/Using-Fleet/Adding-hosts.md

```diff
@@ -36,14 +36,14 @@ Check out the example below:
 
 The above command should be run on a macOS device as notarizing and signing of macOS osquery installers can only be done on macOS devices.
 
-Also remember to replace both `AC_USERNAME` and `AC_PASSWORD` environment variables with your Apple ID and a valid [app-specific](https://support.apple.com/en-ca/HT204397) password respectively.
+Also, remember to replace both `AC_USERNAME` and `AC_PASSWORD` environment variables with your Apple ID and a valid [app-specific](https://support.apple.com/en-ca/HT204397) password, respectively.
 
 ### Fleet Desktop
 
 How to install Fleet Desktop on your hosts:
 1. On the top bar in the Fleet UI, select **Hosts > Add hosts**.
 2. Select the **Include Fleet Desktop** checkbox.
-3. Select the clipboard icon to copy the `fleetctl package` command.
+3. Select the clipboard icon to copy the `fleetctl package` command. 
```

Comment:
Can "the clipboard icon" be the actual icon?

Reply:
I had the same thought 🧠 

What do you think about capitalizing and bolding the icon name and included the icon in parentheses? This means, "clipboard icon" would become "**Clipboard** (insert-clipboard-icon-here) icon"

This seems to be what GitLab uses in their docs:
![Screen Shot 2022-06-10 at 2 37 00 PM](https://user-images.githubusercontent.com/47070608/173154449-be9086a8-ff62-401d-93d5-f6116835984b.png)

The above is taken from this GitLab doc page: https://docs.gitlab.com/ee/user/group/

Reply:
I'm on board. How do I add the icon?

Reply:
> I'm on board. How do I add the icon?

@Desmi-Dizney hmm, it looks like GitLab uses an SVG inline to accomplish this. I think we can make this "adding the icon" improvement later.

For now, I think "the clipboard icon" is good enough.